### PR TITLE
Circleci improvements

### PR DESCRIPTION
**Pull request by renatomefi**
# Usabilla PHP Docker Template

Reviewers: @fabiotc @rdohms @WyriHaximus @renatomefi @frankkoornstra @agustingomes @cvmiert

## Type

Please specify the type of changes being proposed:

| Q                 | A
| ----------------- | -----
| Documentation?    | no
| Dockerfile change?| no
| Build feature?    | yes
| Apply CVE Patch?  | no
| Remove CVE Patch? | no

## Changelog

- [x] Build less often
- [x] Add reusable commands
- [x] Only push upon manual confirmation, unless it's the scheduled build

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -83,6 +83,24 @@ workflows:
               only:
                 - master
 
+commands:
```
Comment (frankkoornstra) [inline .circleci/config.yml]: nice

File: .circleci/config.yml [hunk 2]
```diff
@@ -1,80 +1,85 @@
...
+    - test-cli
+    - test-fpm
+    - test-http
+    - scan-vulnerability
+
+.scheduled_workflow_jobs: &scheduled_workflow_jobs
   - lint
   - lint-shell
-  - build-http:
+  - build-http: &build-http
```
Comment (frankkoornstra) [inline .circleci/config.yml]: The next three are all the same and then there's three that are again all the same. Why do we not have one of each and also one alias?

Comment (renatomefi) [reply .circleci/config.yml]: Yeah I want to make it looking better, but I'm suffering with this yaml and circle ci. Will try more later

Comment (fabiotc) [reply .circleci/config.yml]: image

**Pull request merged**